pandas/core/arrays/base.py

```diff
@@ -1076,7 +1077,7 @@ def repeat(self, repeats: int | Sequence[int], axis: int | None = None):
 
     def take(
         self: ExtensionArrayT,
-        indices: Sequence[int],
+        indices: TakeIndexer,
```

Comment:
Since `TakeIndexer` includes an array, and we say "sequence of ints" in the docs, maybe update the docstring to say that an ndarray is allowed.

Reply:
What would be the best phrasing for that?

"array-like"
"sequence of ints, np.ndarray"
"sequence of ints, one-dimensional np.ndarray of ints"